### ISSUE TITLE
Round to next year if previous round was 12 months.

### ISSUE THREAD

**Issue by lolobosse** [EvanHahn/HumanizeDuration.js#88]
Hi there,

I tested the lib with some random data and I think I find a small bug.

Here is my code:

```javascript
de_humanizer = humanizeDuration.humanizer({
      language: "de",
      units: ["y", "mo"]
    });
var time = 3692131200000;
var duration = de_humanizer(time, {round: true});
```

And duration is `116 Jahre, 12 Monate` which is a bit incoherent I guess.

Great work though!

Best

**Comment by EvanHahn** [comment]
Good catch! I'll fix this.

**Comment by EvanHahn** [comment]
This should be fixed in `humanize-duration@3.6.1`. Let me know if you run into more issues?
